### PR TITLE
prettify status

### PR DESCRIPTION
**Pull request by SimonLab**
see #200

### CODE REVIEW

File: lib/helpers/json-to-html-data/pretty-status.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = function (status) {
+  var mapping = {
+    'permanent collection': 'Permanent collection',
+    'loan: wellcome trust': 'Loan: Wellcome Trust',
+    'loans: historic and loan': 'Loan'
```
Comment (Danwhy) [inline lib/helpers/json-to-html-data/pretty-status.js]: I think here, what Emily meant in https://github.com/TheScienceMuseum/collectionsonline/issues/200#issuecomment-259397680 was that 'loans: historic' and 'loan' should both be 'Loan'.  

See http://localhost:8000/objects/co62406/receiver-unit-from-the-eniac-computer-1943-1945-computer-component and http://localhost:8000/objects/co30866/ora-wireless-receiving-triode-1921-thermionic-valve for examples

Comment (SimonLab) [inline lib/helpers/json-to-html-data/pretty-status.js]: yes I should have look a bit better at the issue, it's true that "historic and loan" as a value doesn't make any sense, thanks @Danwhy

**Comment by Danwhy** [comment]
LGTM :+1: Tres joli

**Comment by nelsonic** [comment]
@SimonLab code looks good. please resolve merge conflicts and re-assign. thanks! 👍

**Comment by nelsonic** [comment]
@SimonLab thanks. merging. 🎉

**Pull request merged**